### ISSUE TITLE
Padding Issue with ScrollOverflow

### ISSUE THREAD

**Issue by meceware** [alvarotrigo/fullPage.js#3335]
Hi Alvaro,

When paddingTop is used with scrolloverflow, a bottom padding occurs on window resize.

### Link to isolated reproduction with no external CSS / JS
https://codepen.io/anon/pen/bjobOY

### Steps to reproduce it
1. Open the link.
2. Check the first section scrollbar position at the end. It's at the end of the section.
3. Resize the browser.
4. Check the first section scrollbar. There is a padding at the end of the section.

### Versions 
Chrome, Firefox

Thank you,

**Comment by alvarotrigo** [comment]
Thanks for reporting it!
I'll take a look at it!

**Comment by alvarotrigo** [comment]
Fixed on the dev branch! To be merged in the next release.

**Comment by meceware** [comment]
Hi Alvaro,

There might be an error here. On resize, it gives an error as

```
scrolloverflow.js:2529 Uncaught ReferenceError: options is not defined
    at Object.update (VM67 scrolloverflow.js:2529)
    at createScrollBar (VM67 scrolloverflow.js:2268)
    at VM67 scrolloverflow.js:2305
    at NodeList.forEach (<anonymous>)
    at forEachSectionAndSlide (VM67 scrolloverflow.js:2297)
    at scrollBarHandler.createScrollBarForAll (VM67 scrolloverflow.js:2227)
    at reBuild (VM47 fullpage.js:483)
    at VM47 fullpage.js:2426
```

The error is not available on the master branch scrolloverflow.js but it can be seen via dev branch scrolloverflow.js.

https://codepen.io/anon/pen/YORrrN

Steps to reproduce it
- Resize the window.

Thank you,

**Comment by alvarotrigo** [comment]
@meceware thanks for reporting it! Kinda weird I do not get the error message on the console on my computer using my own testing page with your exact same setup. 

However, this last commit should fix it. Let me know if it is all ok now.

**Comment by meceware** [comment]
@alvarotrigo Yeah that's fixed now. I was going to suggest using iscrollHandler.fullpageOptions but I thought you were thinking something else with the options.

Thank you,

**Comment by alvarotrigo** [comment]
@meceware 😄 glad you though to solve it in the same way! 

Btw, gonna make a new release by the end of this week! Enough fixed bugs to justify it!

**Comment by meceware** [comment]
@alvarotrigo Looking forward to it :)

**Comment by alvarotrigo** [comment]
Fixed in the latest version 3.0.3
